src/main/java/de/tum/in/www1/artemis/service/connectors/bitbucket/dto/BitbucketCloneDTO.java

```diff
@@ -45,5 +57,9 @@ public String getKey() {
         public void setKey(String key) {
             this.key = key;
         }
+
+        public String toString() {
+            return String.format("project={key=%s}", key);
```

Comment:
please avoid String.format and use normal string concatenation
check out other toString() methods, you can actually generate those automatically in IntelliJ

Reply:
thanks for the tip with IntelliJ